retromock/src/main/java/co/infinum/retromock/Retromock.java

```diff
@@ -223,8 +215,59 @@ private Retromock(final Retrofit retrofit,
     }
     final T delegate = factory.create();
 
-    return (T) Proxy.newProxyInstance(service.getClassLoader(), new Class[] {service},
+    return (T) Proxy.newProxyInstance(service.getClassLoader(), new Class[]{service},
       new InvocationHandler() {
+
+        private boolean isKotlinSuspendFunction;
+        private boolean continuationWantsResponse;
+
+        private Object handleContinuationWithResponse(Object[] args, Object call) {
+          //noinspection unchecked Checked by reflection inside RequestFactory.
```

Comment:
There is no RequestFactory in Retromock

Reply:
Leftover when pasting stuff from Retrofit :D I'll remove the comment.